subiquity/server/controllers/mirror.py

```diff
@@ -65,9 +66,9 @@ async def apply_autoinstall_config(self, context):
         except asyncio.TimeoutError:
             pass
 
-    def on_countrycode(self, cc):
+    def on_geoip(self):
```

Comment:
This reminds me of the fact that I think I made a wrong call in MessageHub: you can call subscribe with arguments that will be passed to the callback, but I think it would be better to allow arguments to be passed to broadcast, which can then be passed to subscribers. Then the GEOIP message could be sent with the geoip data. Just a thought, certainly doesn't have to be changed in this branch.

Reply:
Yes, I debated adding that or not.  Follow-on branch.